kpe/src/pipeline.rs

```diff
@@ -26,9 +28,32 @@ pub enum PipelineError {
     Tokenizer(#[from] TokenizerError),
     /// Failed to run the model: {0}
     Model(#[from] ModelError),
+    /// Failed to load binary parameters from a file: {0}
+    BinParams(#[from] LoadingBinParamsFailed),
```

Comment:
Can you please also add the two errors here and map them in the `from` method 

```
    /// Failed to build the tokenizer: {0}
    TokenizerBuild(#[source] TokenizerError),
    /// Failed to build the model: {0}
    ModelBuild(#[source] ModelError),
```

it will make easier to debug